library/src/main/java/com/chuckerteam/chucker/internal/support/TeeSource.kt

```diff
@@ -27,6 +27,7 @@ internal class TeeSource(
     private var totalBytesRead = 0L
     private var reachedLimit = false
     private var upstreamFailed = false
+    private var upstreamConsumed = false
```

Comment:
Can we change the name here to something more usual for boolean variables - `isUpstreamConsumed`?
Also, just saw that 2 variables above could also be renamed.